extractor/src/extractors/GoModExtractor.ts

```diff
@@ -84,7 +84,8 @@ export default class GoModExtractor implements Extractor {
                         }
                     }
                     break;
-
+                
+                case "retract":
```

Comment:
We should add an associated test case for this as well to ensure the line is parsed properly.

Reply:
I agree. However I'm having trouble running the extractor test cases on my local machine by doing `npm test` in the extractor directory. The test cases run but a couple fail. I also tried building and running the Dockerfile.dev in the same directory and I got the same results. Is there something that I'm not doing correctly? 

Once I figure this out I'll look more into the test cases.

Reply:
I ran the test cases with `npm test -- -u` instead and they seem to work now. I'll get to work on this.

Reply:
Realized that I was updating the Snapshots so of course it worked... 

Now I'm getting an empty array of dependencies for GoModExtractor, even with the `retract` case removed.